### PR TITLE
Adapt GitHub new issue template

### PR DESCRIPTION
**Pull request by LiuHaiYang**
### Why are these changes needed?
Adapt Github new issue form template, preview: https://github.com/LiuHaiYang/incubator-linkis/issues/new/choose

### Related issue number
#1061 

### Checks
---
linkis Component
...

### CODE REVIEW

**Comment by LiuHaiYang** [comment]
@peacewong @wushengyeyouya @clay4444 Can you help to review this pr?

**Comment by peacewong** [comment]
> @peacewong @wushengyeyouya @clay4444 Can you help to review this pr?

ok Thanks!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question or get support
+    url: https://github.com/apache/incubator-linkis/issues
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/config.yml:4]: the URL should refer to the discussion page, right?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        - label: >
+            I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar
+            issues.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Linkis Component
+      description: |
+        What component are you using? Linkis has many subprojects, please make sure to choose the component that
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/bug_report.yml:29]:
```suggestion
        What component are you using? Linkis has many modules, please make sure to choose the modul that
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        you found the bug.
+      multiple: true
+      options:
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]: I think these modules listed here aren't exactly correct, Could you help to confirm this?  @peacewong

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      label: Are you willing to submit a PR?
+      description: >
+        This is absolutely not required, but we are happy to guide you in the contribution process
+        especially if you already have a good understanding of how to implement the fix.
+      options:
+        - label: Yes I am willing to submit a PR!
+
+  - type: markdown
+    attributes:
+      value: "Thanks for completing our form!"
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/bug_report.yml:99]: new line

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+    about: Ask a question or request support for using Linkis
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/config.yml:5]: new line

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      label: Are you willing to submit a PR?
+      description: >
+        This is absolutely not required, but we are happy to guide you in the contribution process
+        especially if you already have a good understanding of how to implement the feature.
+      options:
+        - label: Yes I am willing to submit a PR!
+
+  - type: markdown
+    attributes:
+      value: "Thanks for completing our form!"
```
Comment (clay4444) [inline .github/ISSUE_TEMPLATE/feature_request.yml:63]: new line

**Review by clay4444 — CHANGES_REQUESTED** [review]
It's great overall!  left a few small comments

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (peacewong) [reply .github/ISSUE_TEMPLATE/bug_report.yml:32]: Yes, it should only contain the following modules:
```
linkis-commons
linkis-cg-entrance
linkis-cg-manager
linkis-cg-engineConnplugin
linkis-cg-engineConnManager
linkis-cg-client
linkis-extensions
linkis-orchestrator
linkis-public-enhancements
linkis-service-gateway
linkis-service-discovery
web
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
Comment (peacewong) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Hello, this should be configured as false, and Create a blank issue should not be allowed

**Review by peacewong — APPROVED** [review]
LGTM.

**Comment by peacewong** [comment]
@LiuHaiYang @clay4444 Thank you

**Pull request merged**